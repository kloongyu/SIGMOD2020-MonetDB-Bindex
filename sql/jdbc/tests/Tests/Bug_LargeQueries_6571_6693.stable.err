stderr of test 'Bug_LargeQueries_6571_6693` in directory 'sql/jdbc/tests` itself:


# 18:48:54 >  
# 18:48:54 >  "mserver5" "--debug=10" "--set" "gdk_nr_threads=0" "--set" "mapi_open=true" "--set" "mapi_port=34708" "--set" "mapi_usock=/var/tmp/mtest-15166/.s.monetdb.34708" "--set" "monet_prompt=" "--forcemito" "--dbpath=/export/scratch1/dinther/INSTALL/var/MonetDB/mTests_sql_jdbc_tests" "--set" "embedded_c=true"
# 18:48:54 >  

# builtin opt 	gdk_dbpath = /export/scratch1/dinther/INSTALL/var/monetdb5/dbfarm/demo
# builtin opt 	monet_prompt = >
# builtin opt 	monet_daemon = no
# builtin opt 	mapi_port = 50000
# builtin opt 	mapi_open = false
# builtin opt 	mapi_autosense = false
# builtin opt 	sql_optimizer = default_pipe
# builtin opt 	sql_debug = 0
# cmdline opt 	gdk_nr_threads = 0
# cmdline opt 	mapi_open = true
# cmdline opt 	mapi_port = 34708
# cmdline opt 	mapi_usock = /var/tmp/mtest-15166/.s.monetdb.34708
# cmdline opt 	monet_prompt = 
# cmdline opt 	gdk_dbpath = /export/scratch1/dinther/INSTALL/var/MonetDB/mTests_sql_jdbc_tests
# cmdline opt 	embedded_c = true
#main thread:!ERROR:MALException:client.quit:Server stopped

# 18:48:54 >  
# 18:48:54 >  "./Bug_LargeQueries_6571_6693.SQL.sh" "Bug_LargeQueries_6571_6693"
# 18:48:54 >  


# 18:48:54 >  
# 18:48:54 >  java Bug_LargeQueries_6571_6693 "jdbc:monetdb://catskill:34708/mTests_sql_jdbc_tests?user=monetdb&password=monetdb"
# 18:48:54 >  


# 18:48:58 >  
# 18:48:58 >  "Done."
# 18:48:58 >  

