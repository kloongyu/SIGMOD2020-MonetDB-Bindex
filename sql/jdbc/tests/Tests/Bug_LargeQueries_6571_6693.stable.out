stdout of test 'Bug_LargeQueries_6571_6693` in directory 'sql/jdbc/tests` itself:


# 18:48:54 >  
# 18:48:54 >  "mserver5" "--debug=10" "--set" "gdk_nr_threads=0" "--set" "mapi_open=true" "--set" "mapi_port=34708" "--set" "mapi_usock=/var/tmp/mtest-15166/.s.monetdb.34708" "--set" "monet_prompt=" "--forcemito" "--dbpath=/export/scratch1/dinther/INSTALL/var/MonetDB/mTests_sql_jdbc_tests" "--set" "embedded_c=true"
# 18:48:54 >  

# MonetDB 5 server v11.33.4 (hg id: b99c237c4372+)
# This is an unreleased version
# Serving database 'mTests_sql_jdbc_tests', using 4 threads
# Compiled for x86_64-unknown-linux-gnu/64bit with 128bit integers
# Found 31.305 GiB available main-memory.
# Copyright (c) 1993 - July 2008 CWI.
# Copyright (c) August 2008 - 2019 MonetDB B.V., all rights reserved
# Visit https://www.monetdb.org/ for further information
# Listening for connection requests on mapi:monetdb://catskill.da.cwi.nl:34708/
# Listening for UNIX domain connection requests on mapi:monetdb:///var/tmp/mtest-15166/.s.monetdb.34708
# MonetDB/GIS module loaded
# MonetDB/SQL module loaded

Ready.

# 18:48:54 >  
# 18:48:54 >  "./Bug_LargeQueries_6571_6693.SQL.sh" "Bug_LargeQueries_6571_6693"
# 18:48:54 >  


# 18:48:54 >  
# 18:48:54 >  java Bug_LargeQueries_6571_6693 "jdbc:monetdb://catskill:34708/mTests_sql_jdbc_tests?user=monetdb&password=monetdb"
# 18:48:54 >  

Created table: tbl6693 Inserting rows. 3 rows inserted
Script size is 296607
First test repeat 10 times
Iteration: 1 2 3 4 5 6 7 8 9 10 
Completed first test
Second test repeat 10 times
Iteration: 1 2 3 4 5 6 7 8 9 10 
Completed second test
Script size is 112838
Third test repeat 3 times
Iteration: 1 2 3 
Completed third test
Cleanup TABLE tbl6693
Test completed without hanging


# 18:48:58 >  
# 18:48:58 >  "Done."
# 18:48:58 >  

